### PR TITLE
WSO-435 - Resolve Errors - ucgy_programs

### PR DESCRIPTION
**Pull request by SandyVallee**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module [hunk 1]
```diff
@@ -305,55 +303,45 @@ function ucgy_programs_form_alter(&$form, &$form_state, $form_id)
 
         $i = 0;
         foreach($form['field_program_requirements'][LANGUAGE_NONE] as $field_program_requirement_value) {
+          if( isset($field_program_requirement_value['field_program_requirement_set'])) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module]: Remove extra space "if (  isset"

Comment (SandyVallee) [reply sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module]: done

File: sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module [hunk 2]
```diff
@@ -194,11 +194,11 @@ function ucgy_programs_admin_config_form_labels($node, &$form_state)
  * @param $form_state
  */
 function ucgy_programs_admin_config_form_submit($form, &$form_state){
-    if (isset($form_state['values'])) {
-        if (isset($form_state['values'][PROGRAM_REQUIREMENTS_MAINT_MODE_FLAG])) {
+    if (isset ($form_state['values'])) {
```
Comment (taeram) [inline sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module]: Should be `if (isset($form_state['values'])) {`, with no space between `isset` and `(`

File: sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module [hunk 3]
```diff
@@ -194,11 +194,11 @@ function ucgy_programs_admin_config_form_labels($node, &$form_state)
...
+        if (isset ($form_state['values'][PROGRAM_REQUIREMENTS_MAINT_MODE_FLAG])) {
```
Comment (taeram) [inline sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module]: Same as above

File: sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module [hunk 4]
```diff
@@ -194,11 +194,11 @@ function ucgy_programs_admin_config_form_labels($node, &$form_state)
...
             variable_set(PROGRAM_REQUIREMENTS_MAINT_MODE_FLAG, $form_state['values'][PROGRAM_REQUIREMENTS_MAINT_MODE_FLAG]);
         }
-        if (isset($form_state['values'][PROGRAM_REQUIREMENTS_MAINT_MODE_TEXT])) {
+        if (isset ($form_state['values'][PROGRAM_REQUIREMENTS_MAINT_MODE_TEXT])) {
```
Comment (taeram) [inline sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module]: Same as above

File: sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module (hunk 2 above)
Comment (SandyVallee) [reply sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module]: done

File: sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module (hunk 4 above)
Comment (SandyVallee) [reply sites/all/modules/ucalgary/ucgy_program/ucgy_programs/ucgy_programs.module]: done

**Pull request merged**